### ISSUE TITLE
WPML issue - Fatal Error: Argument 1 passed to Twig_Environment::addFunction() must be an instance of Twig_Function, instance of Twig_SimpleFunction given

### ISSUE THREAD

**Issue by luism-s** [timber/timber#1417]
*Here we go again*

There seems to be a issue that is revealed when WPML Translation Management is installed while using Timber 1.3. A fatal error is shown saying that instead of `Twig_SimpleFunction`, `Twig_Function` should be used in several lines of code. 

I then tried replacing the code in FunctionWrapper: 68 and all occurrences in Twig.php, then the site was working again.

Can you check this and try to use Twig_Function in those files?

### Expected behavior
Expected to open the website I'm developing without any errors

### Actual behavior
```
Fatal error: Uncaught TypeError: Argument 1 passed to Twig_Environment::addFunction() must be an instance of Twig_Function, instance of Twig_SimpleFunction given, called in /var/www/project/wp-content/themes/project/vendor/timber/timber/lib/FunctionWrapper.php on line 70 and defined in /var/www/project/wp-content/themes/project/vendor/twig/twig/lib/Twig/Environment.php:805 
Stack trace: 
#0 /var/www/project/wp-content/themes/project/vendor/timber/timber/lib/FunctionWrapper.php(70): Twig_Environment->addFunction(Object(Twig_SimpleFunction)) 
#1 /var/www/project/wp-includes/class-wp-hook.php(298): Timber\FunctionWrapper->add_to_twig(Object(Twig_Environment)) 
#2 /var/www/project/wp-includes/plugin.php(203): WP_Hook->apply_filters(Object(Twig_Environment), Array) 
#3 /var/www/project/wp-content/themes/project/vendor/timber/timber/lib/Twig.php(219): apply_filters('timber/twig', Object(Twig_Environment)) 
#4 /var/www/project/wp-includes/class-wp-hook.php(298): Timber\Twig->add_timber_filters(Object(Twig_Environment)) 
#5 /var/www/ver in /var/www/project/wp-content/themes/project/vendor/twig/twig/lib/Twig/Environment.php on line 805
```

### Steps to reproduce behavior
* Install Timber v1.3.2
* Install Sitepress-Multilingual-CMS plugin (WPML) v3.7.0
* Install WPML Translation Management v2.3.0

### What version of WordPress, PHP and Timber are you using?
WordPress 4.7.4, PHP 7.0, Timber 1.3.0, Sitepress-Multilingual-CMS 3.7.0, WPML Translation Management 2.3.0

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Installed Timber with Composer in my theme folder, and WPML plugins by placing them in wp-content/plugins.

**Comment by luism-s** [comment]
Update: 
This issue persists on Timber 1.3.2

**Comment by jarednova** [comment]
Frak. Frak. Frak. Frak. Frak.
... looking into it again now...

**Comment by luism-s** [comment]
Some info I got from Twig documentation on version 2

````
Functions
As of Twig 1.x, use Twig_SimpleFunction to add a function. The following classes and interfaces will be removed in 2.0:
Twig_FunctionInterface
Twig_FunctionCallableInterface
Twig_Function
Twig_Function_Function
Twig_Function_Method
Twig_Function_Node
As of Twig 2.x, the Twig_SimpleFunction class is deprecated and will be removed in Twig 3.x (use Twig_Function instead). In Twig 2.x, Twig_SimpleFunction is just an alias for Twig_Function.

````

**Comment by luism-s** [comment]
Issue solved with PR #1464
